file-formats/sfpf/examples/z_aff_example_sfpf.sfpf.json

```diff
@@ -0,0 +1,13 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example SFPF for ABAP file formats",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "cloudDevelopment"
+  },
+  "name": "TEMPLATE_NAME",
+  "description": "Description",
```

Comment:
First question:

Is `name` the object's name and `description` the object's description?